### PR TITLE
Bau fix admin failures

### PR DESCRIPTION
**Pull request by bhavanibandla**
Jira link

- BAU

What?
I have added/removed/altered:

-  fixes admin tool CRUD test and lint errors.

Why?
I am doing this because:

- To make sure the build is green and that we have proper coverage

### CODE REVIEW

**Review by jebw — CHANGES_REQUESTED** [review]
test code left in place

File: cypress/e2e/smoketests/smokeTestCI.cy.js [hunk 1]
```diff
@@ -118,19 +130,19 @@ describe('Smoke tests to cover basic functionality', {tags: ['smokeTest']}, func
...
       cy.get('.govuk-table__row').contains('057140');
     });
     it('Quotas Search - Commodity Code', function() {
       cy.visit('/quota_search');
       cy.contains('Search for quotas');
-      cy.get('input#goods_nomenclature_item_id')
-          .click()
-          .clear()
-          .type('3920000000');
+      cy.get('input#goods_nomenclature_item_id').click();
```
Comment (willfish) [inline cypress/e2e/smoketests/smokeTestCI.cy.js:143]: Not sure I understand this change. Is it related to a cypress upgrade?

**Review by willfish — APPROVED** [review]
LGTM

File: cypress/e2e/smoketests/smokeTestCI.cy.js (hunk 1 above)
Comment (bhavanibandla) [reply cypress/e2e/smoketests/smokeTestCI.cy.js:143]: if we use both in the same line - it's throwing lint issues that's why I had to separate it.

**Review by jebw — APPROVED** [review]

**Pull request merged**